dvc/scm/git/__init__.py

```diff
@@ -200,10 +200,10 @@ def add(self, paths):
             self.repo.index.add(paths)
         except AssertionError:
             msg = (
-                "failed to add '{}' to git. You can add those files"
-                " manually using 'git add'."
-                " See 'https://github.com/iterative/dvc/issues/610'"
-                " for more details.".format(str(paths))
+                "failed to add '{}' to git. You can add those files "
+                "manually using 'git add'. See "
+                "https://github.com/iterative/dvc/issues/610 for more "
+                "details.".format(str(paths))
```

Comment:
Removed `'` aroud URL and moved trailing spaces to the end of strings, rewrapped.